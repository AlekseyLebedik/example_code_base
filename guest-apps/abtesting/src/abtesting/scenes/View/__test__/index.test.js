import React from 'react';
import { shallow } from 'enzyme';

import ViewForm from '../index';

describe('ABTesting ViewForm', () => {
  it('renders default values', () => {
    expect(shallow(<ViewForm />)).toMatchSnapshot();
  });
});
