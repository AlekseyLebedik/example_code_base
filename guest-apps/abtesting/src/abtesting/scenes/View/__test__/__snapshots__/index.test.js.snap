// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ABTesting ViewForm renders default values 1`] = `
<withRouter(Connect(UpdateTestForm))
  disableFormContext={true}
  mode="VIEW"
  showActionsPanel={true}
  viewOnly={true}
/>
`;
